app/helpers/data_sharing_agreement_helper.rb

```diff
@@ -1,7 +1,7 @@
 module DataSharingAgreementHelper
   def data_sharing_agreement_row(user:, organisation:, summary_list:)
     summary_list.with_row do |row|
-      row.with_key { "Data Sharing Agreement" }
+      row.with_key { "Data protection agreement" }
```

Comment:
Any reason for renaming it/is that the correct casing?

Reply:
It's just what is in the designs. I have messaged the higher ups to double check.